pyqtgraph/graphicsItems/TargetItem.py

```diff
@@ -106,19 +106,35 @@ def __init__(
             pos = Point(0, 0)
         self.setPos(pos)
 
+        self._path = None
+        self.setSymbol(symbol)
+
+        self.scale = size
+        self.setLabel(label, labelOpts)
+
+    def setSymbol(self, symbol):
+        r"""Method to set the TargetItem symbol, during or after creation
```

Comment:
Fairly sure this does not need to be a raw string.

Reply:
Also this docstring should probably have the `Raises` section to specify the KeyError that is raised if the symbol isn't found.

Reply:
I made it a raw string since that seemed to be how some of the other docstrings were formatted and was for consistency, I've made these changes in my latest commit.